scalding-commons/src/main/java/com/twitter/scalding/commons/datastores/VersionedStore.java

```diff
@@ -152,7 +152,6 @@ public void cleanup(int versionsToKeep) throws IOException {
             // have both version suffix and success flag below
             Set<Long> ret = new HashSet<Long>();
             for(Path p: listDir(getRoot())) {
```

Comment:
Note you can use listStatus to avoid additional fileStatus RPC later on for directory contents.